terminus-ui/src/validation-messages/validation-message.service.ts

```diff
@@ -36,13 +36,20 @@ export class TsValidationMessageService {
    * @param validatorName - The name of the validator
    * @param validatorValue - The value of the validator
    * @return The error message
+   *
+   * @deprecated target 11.x
```

Comment:
We should move this deprecation notice to comments just above the items being deprecated (in this case, specific validators). This currently seems like we will be deprecating this entire method.